packages/main/src/themes/PopupsCommon.css

```diff
@@ -44,13 +44,17 @@
     margin: 0;
     color: var(--sapPageHeader_TextColor);
     font-size: 1rem;
-    font-weight: var(--_ui5_popup_header_footer_font_weight);
     font-family: "72override", var(--sapFontFamily);
     display: flex;
     justify-content: center;
     align-items: center;
 }
 
+.ui5-popup-header-root .ui5-popup-header-text,
+.ui5-popup-footer-root .ui5-popup-header-text {
```

Comment:
It looks fine for the header, I just wonder do we have "ui5-popup-header-text" in the footer ".ui5-popup-footer-root"